test/runtests.jl

```diff
@@ -2178,4 +2178,6 @@ end
 
 using Aqua
 
-Aqua.test_all(Unitful, ambiguities=VERSION≥v"1.1", unbound_args=false, piracy=VERSION≥v"1.8")
+Aqua.test_all(Unitful, ambiguities=VERSION≥v"1.1", unbound_args=false, piracy=VERSION≥v"1.8", stale_deps=false, project_toml_formatting=VERSION≥v"1.6")
+# Workaround for https://github.com/JuliaTesting/Aqua.jl/issues/107:
+Aqua.test_stale_deps(Unitful, ignore=[:InverseFunctions])
```

Comment:
```suggestion
Aqua.test_all(Unitful, ambiguities=VERSION≥v"1.1", unbound_args=false, piracy=VERSION≥v"1.8", project_toml_formatting=VERSION≥v"1.6")
```
https://github.com/JuliaTesting/Aqua.jl/issues/107 seems to be fixed in Aqua v0.6.3, so maybe we can simplify this and restrict the compat entry for Aqua.